### PR TITLE
Improve error printing in contract validator

### PR DESCRIPTION
**Pull request by SupunS**
Depends on #5600

Looks like without providing the codes, the secondary errors are not getting printed.

### CODE REVIEW

**Comment by turbolent** [comment]
@SupunS Is this PR only in draft because it depends on another PR, or is there still other work required?

**Review by turbolent — APPROVED** [review]
👍

**Comment by SupunS** [comment]
Was in draft until the flow-go gets updated withe cadence release

File: cmd/util/ledger/migrations/staged_contracts_migration_test.go [hunk 1]
```diff
@@ -1505,3 +1506,185 @@ func TestConcurrentContractUpdate(t *testing.T) {
...
+		contractACode := newContractPayload(common.Address(address), "A", []byte(oldCodeA))
+		nftCode := newContractPayload(nftAddress, "NonFungibleToken", []byte(nftContract))
+
+		accountPayloads := []*ledger.Payload{contractACode}
+		allPayloads := []*ledger.Payload{contractACode, nftCode}
+
+		err = migration.InitMigration(log, allPayloads, 0)
+		require.NoError(t, err)
+
+		accountPayloads, err = migration.MigrateAccount(
```
Comment (turbolent) [inline cmd/util/ledger/migrations/staged_contracts_migration_test.go:1666]:
```suggestion
		_, err = migration.MigrateAccount(
```

**Pull request merged**